contentcuration/contentcuration/frontend/channelEdit/components/sidePanels/SubmitToCommunityLibrarySidePanel/index.vue

```diff
@@ -398,6 +424,37 @@
         return channelVersion;
       });
 
+      const {
+        isLoading: licenseAuditIsLoading,
+        isFinished: licenseAuditIsFinished,
+        invalidLicenses,
+        specialPermissions,
+        includedLicenses,
+        checkAndTriggerAudit: checkAndTriggerLicenseAudit,
+      } = useLicenseAudit(toRef(props, 'channel'), currentChannelVersion);
+
+      const allSpecialPermissionsChecked = ref(true);
+
+      const hasInvalidLicenses = computed(() => {
+        return invalidLicenses.value && invalidLicenses.value.length > 0;
+      });
+
+      const canBeSubmitted = computed(() => {
+        if (!allSpecialPermissionsChecked.value) return false;
+        if (isPublishing.value) return false;
+        if (hasInvalidLicenses.value) return false;
+        if (!licenseAuditIsFinished.value) return false;
+
+        const baseCondition =
+          canBeEdited.value && publishedDataIsFinished.value && description.value.length >= 1;
+
+        if (needsReplacementConfirmation.value) {
+          return baseCondition && replacementConfirmed.value;
+        }
+
+        return baseCondition;
+      });
```

Comment:
Now that we are adding each time more conditions so that `canBeSubmitted` is true, we can replace all of this with an array of conditions that needs to be met to return true. This way, we make the conditions to be met more explicit:

```JavaScript
const conditions = [
    allSpecialPermissionsChecked.value,
    !isPublishing.value,
    !hasInvalidLicenses.value,
    licenseAuditIsFinished.value,
    canBeEdited.value,
    publishedDataIsFinished.value,
    description.value.length >= 1,
];

if (needsReplacementConfirmation.value) {
  conditions.push(replacementConfirmed.value);
}

return conditions.every(condition => condition);
```